driver-core/src/main/com/mongodb/ReadPreferenceHedgeOptions.java

```diff
@@ -28,15 +28,15 @@
  */
 @Immutable
 public final class ReadPreferenceHedgeOptions {
-    private final boolean hedgedReads;
+    private final boolean isEnabled;
 
     /**
      * Gets whether hedged reads are enabled in the server.
      *
      * @return true if hedged reads are enabled in the server
      */
-    public boolean getHedgedReads() {
-        return hedgedReads;
+    public boolean getEnabled() {
```

Comment:
The convention in Java is to prefix getters for boolean properties with "is", so this should be "isEnabled"